File: Python/1066.py
numeros = []
pos = 0
imp = 0
par = 0
neg = 0

for i in range(5):
    n = int(input())
    numeros.append(n)
    
for x in numeros:
    if x > 0:
        pos += 1
    elif x < 0:
        neg += 1
    if x %2 == 0:
        par += 1
    elif x%2 == 1:
        imp += 1
        
print(f'{par} valor(es) par(es)')
print(f'{imp} valor(es) impar(es)')
print(f'{pos} valor(es) positivo(s)')
print(f'{neg} valor(es) negativo(s)')

File: C/1066.c
#include <stdio.h>
 
int main() {
 
    int par = 0, impar = 0, positivo = 0, negativo = 0;
    
    for(int i = 0; i < 5; i++){
        int num;
        scanf("%d", &num);
        
        if(num % 2){
            impar += 1;
        }else{
            par += 1;
        }
        if(num < 0){
            negativo += 1;
        }else if(num > 0){
            positivo += 1;
        }
    }
    
    printf("%d valor(es) par(es)\n", par);
    printf("%d valor(es) impar(es)\n",impar);
    printf("%d valor(es) positivo(s)\n", positivo);
    printf("%d valor(es) negativo(s)\n",negativo); 
} 
